tests/SemVerAnalyzer.Tests/IntegrationTests.cs

```diff
@@ -104,5 +104,19 @@ public async Task CompareSameAssembly()
 
 			PAssert.IsTrue(() => report == null);
 		}
+
+		[Fact]
+		public async Task OmitDisclaimer()
+		{
+			var command = new CompareCommand {
+				Assembly = "Local.dll",
+				PackageName = "Major",
+				OmitDisclaimer = true
+			};
+
+			var report = await _runner.Compare(command);
+
+			PAssert.IsTrue(() => !report.Contains("*This is a sanity check"));
```

Comment:
probably needs to be the inverse test too incase the text changes

Reply:
Will do another PR.